### ISSUE TITLE
window is not defined

### ISSUE THREAD

**Issue by forna** [retrotheft/svelte-mainloop#1]
I get the error:

```
ReferenceError: window is not defined
    at ...\node_modules\svelte-mainloop\dist\mainloop.js:112:31
```

I would suggest to add a check to ensure the code referencing window** is running in the browser using something like:
```
if (typeof window !== "undefined") {
```

**Comment by retrotheft** [comment]
Oh dang, thanks for that! I spend so much time using Svelte without SvelteKit that I didn't even think to check. D'oh!

I've released a new version: 1.1.2, that checks for window and passes a dummy loop if window is undefined. It'll log a message on the server console and should work fine on the client side now.

Let me know if you still have any issues! Thanks again :)

**Comment by retrotheft** [comment]
# Fixed with v1.1.2
